specification/schema/content.schema.json

```diff
@@ -18,6 +18,18 @@
             "type": "string",
             "description": "A uri that points to tile content. When the uri is relative, it is relative to the referring tileset JSON file."
         },
+        "metadata": {
+            "allOf": [
+                {
+                    "$ref": "metadataEntity.schema.json"
+                }
+            ],
+            "description": "Content metadata."
```

Comment:
At some point we'll need to flesh out the new properties' descriptions since I hastily put those together the first time around. Doesn't have to be this PR though.